components/mpas-ocean/src/shared/mpas_ocn_surface_land_ice_fluxes.F

```diff
@@ -650,6 +655,17 @@ subroutine ocn_surface_land_ice_fluxes_build_arrays(meshPool, &
 
       endif ! jenkinsOn or hollandJenkinsOn
 
+      ! Add frazil and interface melt/freeze to get total fresh water flux
+      if ( associated(frazilIceFreshwaterFlux) ) then
+          do iCell = 1, nCells
+                landIceFreshwaterFluxTotal(iCell) = landIceFreshwaterFlux(iCell) + landIceFloatingMask(iCell)*frazilIceFreshwaterFlux(iCell)
```

Comment:
There won't be frazil under grounded ice so it probably won't matter but your suggestion is more correct.

Reply:
So I think it is a good idea to keep it consistent with the mask used to calculate accumulatedLandIceFrazilMass, at the moment the mask there is landIceMask:

https://github.com/E3SM-Project/E3SM/blob/dcc0ffb6db72edd3ce1ada559f4c2755d8dce15b/components/mpas-ocean/src/shared/mpas_ocn_frazil_forcing.F#L619

So do you suggest to change that to be modulated to landIceFloatingFraction also?

Reply:
Thanks for pointing me to that code. I see that we shouldn't change to `landIceFloatingFraction` without making other changes as well. Since those changes could be non-BFB, I don't think we should tackle them right now (i.e., don't change anything here).

I think that if we were to change to `landIceFloatingFraction` we would want to account for the fact that open ocean frazil forms at a different salinity than that under land ice. I don't want to make any implicit assumptions that we only have fractional ice cells at the grounding line (i.e., that frazil would form in the non-floating fraction).

@xylar I was actually more concerned with some future scenario where we have fractional ice cells at the calving front (maybe when we have dynamic calving).

Reply:
Ah, sorry, it was late and I was tired. @irenavankova and I already discussed this.  We don't want to use the `landIceFloatingFraciton` in this case.  That handles the difference between floating and grounded ice but doesn't really handle half floating and half open ocean.  The frazil flux that is in such a cell would be lost, since it also doesn't go to the sea-ice model.  So I believe the mask is correct as it is.

Reply:
I mistakenly thought the discussion was about `landIceMask` vs `landIceFloatingMask`.